SpeechResponder/ScriptResolver.cs

```diff
@@ -155,6 +155,17 @@ public string resolveFromValue(string script, BuiltinStore store, bool isTopLeve
                 Logging.Warn($"Failed to resolve {scriptName} at line {e.Line}. {e}");
                 return $"There is a problem with {scriptName} at line {e.Line}. {errorTranslation(e.Message)}";
             }
+            catch (Exception e)
+            {
+                Dictionary<string, object> data = new Dictionary<string, object>
+                {
+                    { "exception", e },
+                    { "script", script },
+                    { "store", JsonConvert.SerializeObject(store) }
```

Comment:
I don't think it's a good idea to include the store. (a) it's an opaque object that doesn't really belong to us (b) it's deprecated in later version of Cottle. Maybe `EDDI.Instance.State` instead?

Reply:
Perhaps. What I really want is the output from the createVariables()` method (which is not currently directly available from the `resolveFromValue()` method, though we could make modifications to pass it.